### PR TITLE
manifest: update homekit revision

### PR DESCRIPTION
**Pull request by mariuszpos**
nrfconnect/sdk-homekit#452

**Above PR for HomeKit requires upmerge changes, so my PR is temporary rebased to: https://github.com/nrfconnect/sdk-nrf/pull/9669
As soon as upmerge will be merged I will rebase this PR to main.**

Signed-off-by: Mariusz Poslinski <mariusz.poslinski@nordicsemi.no>

### CODE REVIEW

**Review by sachinthegreen — APPROVED** [review]

**Comment by kkasperczyk-no** [comment]
@mariuszpos thanks for contributing and aligning the other samples as well. Could you describe a little bit what are the changes introduced by this PR? Looking at the code I guess now some hooks are available in the MCUMGR library, so we should be able to better handle DFU process in our applications. From the Matter perspective please be aware that there are many other samples apart from weather station that may optionally use DFU over SMP, so we should align them as well. I think the best option would be to add `select` for this new mcumgr configs here: https://github.com/nrfconnect/sdk-connectedhomeip/blob/master/config/nrfconnect/chip-module/Kconfig.features#L76. Would you mind adding such change or someone from our team should assist you?

**Review by krish2718 — APPROVED** [review]
For `drivers/wifi/nrf700x/zephyr/src/qspi/src/spi_if.c`

**Comment by kkasperczyk-no** [comment]
@mariuszpos sorry I didn't realize it is not your commit, but you just included upmerge commits :) Please ignore my previous comment.

**Review by rakons — APPROVED** [review]

File: samples/bluetooth/mesh/chat/prj.conf [hunk 1]
```diff
@@ -54,7 +54,6 @@ CONFIG_BT_MESH_DK_PROV=y
 # Enable Bluetooth mesh models debug logs
 CONFIG_BT_DEBUG_LOG=y
 CONFIG_BT_MESH_DEBUG=y
-CONFIG_BT_MESH_DEBUG_MODEL=y
```
Comment (alxelax) [inline samples/bluetooth/mesh/chat/prj.conf:57]: What is the reason for disabling model debugging here but leaving mesh stack debugging?  App demonstrates vendor model implementation.

**Review by alxelax — CHANGES_REQUESTED** [review]

**Review by alxelax — COMMENTED** [review]
I see this came from upmere PR. I'll redirect my questions there.

**Review by alxelax — APPROVED** [review]

**Comment by mariuszpos** [comment]
This PR is no longer required because the homekit revision update has been added to the PR: https://github.com/nrfconnect/sdk-nrf/pull/9669

**Pull request closed without merging**